### PR TITLE
Version 2.0

### PR DESCRIPTION
**Pull request by Davidmattei**
|Q              |A  |
|---------------|---|
|Bug fix?       |yes|
|New feature?   |yes|	
|BC breaks?     |no|	
|Deprecations?  |no|	
|Fixed tickets  |no|	

Requires new releases:
- CommonBundle: https://github.com/ems-project/EMSCommonBundle/pull/97
- FormBundle: https://github.com/ems-project/EMSFormBundle/pull/84

### CODE REVIEW

**Comment by dameert** [comment]
Update this line in phpstan: 

        - %currentWorkingDirectory%/DependencyInjection/Configuration.php

To add the /src subfolder

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,3 @@
+.phpunit.result.cache
 composer.lock
-phpunit.xml
```
Comment (dameert) [inline .gitignore:2]: See comment on #32 /phpunit.xml is the local version of /phpunit.xml.dist

Comment (dameert) [inline .gitignore:2]: See comment on #32 /phpunit.xml is the local version of /phpunit.xml.dist

**Review by dameert — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Davidmattei) [reply .gitignore:2]: Fixed in #34

**Comment by Davidmattei** [comment]
96% coverage 👍

File: src/FormConfig/EmailConfig.php [hunk 2]
```diff
@@ -22,8 +22,8 @@ public function __construct(RenderedSubmission $submission)
         $this->endpoint = $submission->getEndpoint();
         $message = \json_decode($submission->getMessage(), true);
 
-        $this->from = $message['from'];
-        $this->subject = $message['subject'];
+        $this->from = $message['from'] ?? 'noreply@elasticms.eu';
```
Comment (dameert) [inline src/FormConfig/EmailConfig.php]: do we want our hosting to be the default email address for everybody using the project?

File: tests/Functional/Handler/EmailHandlerTest.php [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+    public function testSubmitMultipleFiles(): void
+    {
+        $data = ['name' => 'David', 'files' => [
+            new UploadedFile(__DIR__ . '/../../files/attachment.txt', 'attachment.txt', 'text/plain'),
+            new UploadedFile(__DIR__ . '/../../files/attachment2.txt', 'attachment2.txt', 'text/plain'),
+        ]];
+        $form = $this->formFactory->createBuilder(FormType::class, $data, [])
+            ->add('name', TextType::class)
+            ->add('files', FileType::class, ['multiple' => true])
```
Comment (dameert) [inline tests/Functional/Handler/EmailHandlerTest.php]: Should we not use our components in testing in stead of the Symfony components directly?

**Review by dameert — CHANGES_REQUESTED** [review]

File: src/FormConfig/EmailConfig.php (hunk 2 above)
Comment (Davidmattei) [reply src/FormConfig/EmailConfig.php]: Fix https://github.com/ems-project/EMSSubmissionBundle/pull/31/commits/691e4279d3dd93797c55d50ef6d6b6ab49d97ded

File: tests/Functional/Handler/EmailHandlerTest.php (hunk 3 above)
Comment (Davidmattei) [reply tests/Functional/Handler/EmailHandlerTest.php]: No that's the responsibility of the form bundle

**Review by dameert — DISMISSED** [review]

**Review by dameert — APPROVED** [review]

**Pull request merged**